build/build-image/cross/Makefile

```diff
@@ -14,14 +14,15 @@
 
 .PHONY:	build push
 
+REGISTRY?=staging-k8s.gcr.io
 IMAGE=kube-cross
 TAG=$(shell cat VERSION)
 
 
 all: push
 
 build:
-	docker build --pull -t staging-k8s.gcr.io/$(IMAGE):$(TAG) .
+	docker build --pull -t $(REGISTRY)/$(IMAGE):$(TAG) .
 
 push: build
-	docker push staging-k8s.gcr.io/$(IMAGE):$(TAG)
+	docker push $(REGISTRY)/$(IMAGE):$(TAG)
```

Comment:
also, if the image is arch-specific, it probably wants an arch-specific tag?

Reply:
and it's probably best to add the manifest as well!

Reply:
I don't know how relevant to call this image as `kube-cross` for non-amd64 architecture because we aren't installing `crossbuild-essential-<platform>` for the non-amd64 platform so that way I really whether we will be able to build any binaries for different archs other than host arch.